src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -40,10 +42,13 @@ internal class SonarQubeServiceWrapper : ISonarQubeServiceWrapper
         public const string ProjectDashboardRelativeUrl = "dashboard/index/{0}";
 
         public const string RoslynExporter = "roslyn-cs";
-
-        public const string CSharpLanguage = "cs";
-        public const string VBLanguage = "vbnet";
-        public static readonly string[] SupportedLanguages = { CSharpLanguage, VBLanguage };
+        
+        private static readonly IReadOnlyDictionary<Language, string> languageKeys = new ReadOnlyDictionary<Language, string>(
```

Comment:
That was my original assumption and belief.
However because of `LanguageGroup` (and under its previous name `RuleSetGroup`), the strings "CSharp" and "VB" (from the enum.ToString) are now part of the generated solution rule set files, so we still need to capture these IDs somewhere.
I did raise a [ticket](https://jira.sonarsource.com/browse/SVS-54) about the badly structured and named generated files. I don't this this is going to be fixed however; we're currently tightly connected with specific files existing under these assumed paths in order for 'things to work properly', like detection of binding etc.